wasm-host/src/main.rs

```diff
@@ -137,7 +137,7 @@ fn main() {
 
     let data_source = MockData::new(&tx_json, &lo_json, &lh_json, &l_json, &nft_json);
     info!("Executing function: {}", args.function);
-    match vm_wamr::run_func(wasm_file, &args.function, Some(100000), data_source) {
+    match vm_wamr::run_func(wasm_file, &args.function, Some(1000000), data_source) {
```

Comment:
When compiled in debug mode, the contract in this PR exceeds the gas cap. We can either accept this increase in the host simulator, or I can break-up the example into distinct example projects (e.g., one for emitting the current transaction; one for emitting the current escrow object on ledger; and one for emitting an arbitrary account id).

I lean towards the latter, in which case I'll need to re-work this PR.

Reply:
I also increase the gas cap in another PR (not merged). 

No matter what the cap is, I feel we need two caps (imagine release build and debug build). The larger cap is for people to have more room to experiment. The smaller cap matching rippled config is for testing before deploy. @mvadari @intelliot Let me know what you guys think. We don't need it now, of course.